src/gdrdrv/gdrdrv.h

```diff
@@ -27,7 +27,7 @@
 #define GDRDRV_TOSTRING(s)            GDRDRV_STRINGIFY(s)
 
 #define GDRDRV_MAJOR_VERSION    2
-#define GDRDRV_MINOR_VERSION    2
+#define GDRDRV_MINOR_VERSION    3
```

Comment:
has the kernel ABI changed at all?

Reply:
never mind, the implementation has changed enough that we want to store this version in the module itself